less/bootstrap/dropdowns.less

```diff
@@ -195,6 +196,12 @@
   }
 }
 
+.caret-up {
+  .fa-caret-down::before {
+    content: @fa-var-caret-up;
+  }
+}
+
```

Comment:
Do we need to modify dropdowns.less? Best to leave the vendor file alone if possible.

Reply:
Oh, I hadn't thought it as a vendor file. But also hadn't felt quite right. We can put that style any place you see apt.

Reply:
Let's remove the use of `<span class="carat"></span>` in favour of `<i class="fa fa-sort"></i>` instead.